Tests/Functional/Drivers/Queue/PhpAmqpLibDriverTest.php

```diff
@@ -0,0 +1,224 @@
+<?php
+
+namespace Smartbox\Integration\FrameworkBundle\Tests;
```

Comment:
The namespace of this test is not correct in relation to where the file is. I will fix it